src/js/components/icons/status/OK.js

```diff
@@ -2,20 +2,33 @@
 
 var React = require('react');
 var IntlMixin = require('../../../mixins/GrommetIntlMixin');
+var uuid = require('node-uuid');
 
 var OK = React.createClass({
 
   mixins: [IntlMixin],
 
+  propTypes: {
+    a11yTitle: React.PropTypes.string
+  },
+
   render: function() {
     var className = 'status-icon status-icon-ok';
+    //var a11yTitle = this.getGrommetIntlMessage('OK');
+    var a11yTitle;
     if (this.props.className) {
       className += ' ' + this.props.className;
     }
+    if (typeof this.props.a11yTitle === "undefined") {
+      //this.props.a11yTitle = a11yTitle;
+      a11yTitle = this.getGrommetIntlMessage('OK');
```

Comment:
I have attempted to set the default params value like in your comment but would get syntax errors.  Not sure if `getCrommetIntlMessage` is defined yet?